src/ring_theory/ideal/operations.lean

```diff
@@ -119,17 +116,15 @@ theorem mem_smul_span_singleton {I : ideal R} {m : M} {x : M} :
     ⟨y1 + y2, I.add_mem hyi1 hyi2, by rw [add_smul, hy1, hy2]⟩),
 λ ⟨y, hyi, hy⟩, hy ▸ smul_mem_smul hyi (subset_span $ set.mem_singleton m)⟩
 
-theorem smul_le_right : I • N ≤ N :=
-smul_le.2 $ λ r hr n, N.smul_mem r
+theorem smul_le_right : I • N ≤ N := smul_le.2 $ λ r hr n, N.smul_mem r
 
 theorem smul_mono (hij : I ≤ J) (hnp : N ≤ P) : I • N ≤ J • P := map₂_le_map₂ hij hnp
 
 theorem smul_mono_left (h : I ≤ J) : I • N ≤ J • N := map₂_le_map₂_left h
 
 theorem smul_mono_right (h : N ≤ P) : I • N ≤ I • P := map₂_le_map₂_right h
 
-lemma map_le_smul_top (I : ideal R) (f : R →ₗ[R] M) :
-  submodule.map f I ≤ I • (⊤ : submodule R M) :=
+lemma map_le_smul_top (I : ideal R) (f : R →ₗ[R] M) : submodule.map f I ≤ I • (⊤ : submodule R M) :=
```

Comment:
For theorems such as this where you have a couple of arguments and then the return type, and all the important information is in the return type, I'd argue the type deserves its own line even if it fits on the previous one, because it makes finding the type so much easier.
```suggestion
lemma map_le_smul_top (I : ideal R) (f : R →ₗ[R] M) :
  submodule.map f I ≤ I • (⊤ : submodule R M) :=
```